### PR TITLE
[DOC] Document one cause of orphaned chunks in check command, #2295

### PR DESCRIPTION
**Pull request by fantasya-pbem**
Clean up the whole check usage paragraph, use consistent wording.

### CODE REVIEW

File: src/borg/archiver.py [hunk 1]
```diff
@@ -2812,58 +2812,59 @@ def define_borg_mount(parser):
...
-          we try to recover as many objects from the segment as possible.
-        - In repair mode, it makes sure that the index is consistent with the data
-          stored in the segments.
-        - If you use a remote repo server via ssh:, the repo check is executed on the
-          repo server without causing significant network traffic.
+        - In repair mode, if an integrity error is detected in a segment, try to recover
+          as many objects from the segment as possible.
+        - In repair mode, make sure that the index is consistent with the data stored in
+          the segments.
+        - If checking a remote repo via ``ssh:``, the check is executed on the server
```
Comment (ThomasWaldmann) [inline src/borg/archiver.py:2823]: ... the **repo check** ...

File: src/borg/archiver.py [hunk 2]
```diff
@@ -2812,58 +2812,59 @@ def define_borg_mount(parser):
...
-          required).
-        - The archive checks can be time consuming, they can be skipped using the
+          files, check if chunk is present. In repair mode, if a chunk is not present,
+          replace it with a same-size replacement chunk of zeroes. If a previously lost
+          chunk reappears (e.g. via a later backup), in repair mode the all-zero replacement
+          chunk will be replaced by the correct chunk. This requires reading of archive and
+          file metadata, but not data.
+        - In repair mode, when all the archives were checked, orphaned chunks are deleted
+          from the repo. One cause for orphaned chunks are read errors in the archive
+          creation process.
```
Comment (ThomasWaldmann) [inline src/borg/archiver.py:2842]: ... are **input file related errors** in ... (or **source file**?)

(not sure if these are just read errors, i guess all sorts of non-recoverable file-scope errors happening somewhere not right at the beginning lead to that)

Comment (ThomasWaldmann) [reply src/borg/archiver.py:2842]: is it "one cause for" or "one cause of"?

File: src/borg/archiver.py [hunk 3]
```diff
@@ -2812,58 +2812,59 @@ def define_borg_mount(parser):
...
+        - If checking a remote repo via ``ssh:``, the archive check is executed on the
+          client machine. (Because if encryption is enabled, the checks will require
+          decryption, and this is always done client-side as key access is required.)
```
Comment (ThomasWaldmann) [inline src/borg/archiver.py:2845]: i am not a native english speaker, but this way of putting the parens looks strange to me.

```
foo (bar).
```
vs.
```
foo. (bar.)
```

Comment (fantasya-pbem) [reply src/borg/archiver.py:2845]: Well, it's not just "bar" in parentheses, but a full sentence that ends with a full stop. It's a stylistic thing in my opinion. I will find a better phrase.

File: src/borg/archiver.py (hunk 2 above)
Comment (fantasya-pbem) [reply src/borg/archiver.py:2842]: Yes, "one cause of" is better.

BTW, shall I resolve conversations after fixing things, or do you want to this yourself?

File: src/borg/archiver.py (hunk 1 above)
Comment (fantasya-pbem) [reply src/borg/archiver.py]: fixed

File: src/borg/archiver.py (hunk 2 above)
Comment (ThomasWaldmann) [reply src/borg/archiver.py]: doesn't matter, only that it is fixed or discussed.

**Comment by ThomasWaldmann** [comment]
Thanks!

**Pull request merged**